doc/nims.rst

```diff
@@ -6,9 +6,25 @@ Strictly speaking, ``NimScript`` is the subset of Nim that can be evaluated
 by Nim's builtin virtual machine (VM). This VM is used for Nim's compiletime
 function evaluation features.
 
-You can use a ``<myproject>.nims`` file that simply contains Nim code
-controlling the compilation process. For a directory wide
-configuration, use ``config.nims`` instead of ``<myproject>.nims``.
+The ``nim`` executable processes the ``.nims`` configuration files in
+the following directories (in this order; later files overwrite
+previous settings):
+
+1) If environment variable ``XDG_CONFIG_HOME`` is defined,
+   ``$XDG_CONFIG_HOME/nim/config.nims`` or
+   ``~/.config/nim/config.nims`` (POSIX) or
+   ``%APPDATA%/nim/config.nims`` (Windows). This file can be skipped
+   with the ``--skipUserCfg`` command line option.
+2) ``$parentDir/config.nims`` where ``$parentDir`` stands for any
```

Comment:
This hasn't been implemented.

Reply:
You mean that the `--skip*` switches don't apply to .nims files? I then misunderstood the above referenced applied PR.

Reply:
Is this true then?

```rst
1) If environment variable ``XDG_CONFIG_HOME`` is defined,
   ``$XDG_CONFIG_HOME/nim/config.nims`` or
   ``~/.config/nim/config.nims`` (POSIX) or
   ``%APPDATA%/nim/config.nims`` (Windows) will be used.
2) Next, if ``config.nims`` is placed inside a directory, that will
   apply to all Nim projects inside that directory.
3) Next, if a project has ``<project>.nims`` that resides in the same
   directory as the ``<project>.nim``, that will apply finally.
```

Reply:
@Araq  I believe I did implement it, see https://github.com/nim-lang/Nim/pull/8682/files#diff-5fdd8f1b1ac3aa10d48e0af960fb8c17R52
when I pass `--skipUserCfg`, /Users/timothee/.config/nim/config.nims is skipped

Reply:
Never mind, I misremembered then.